examples/finance/bs_ivbp.ipynb

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
<p>You can split the cells to have setup, then run separately (makes it easier when something breaks ot if want to add to CI)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Sorry could you clarify which cells you would like me to split?

Reply:
This one so that you have
cell 1:
```
# Equations with Neumann boundary conditions
eq = [Eq(v[t,extent], v[t,extent-1]+(v[t,extent-1]-v[t,extent-2])),
      Eq(v[t,extent+1], v[t,extent]+(v[t,extent-1]-v[t,extent-2])),
      Eq(v.forward, update_centered)]

op = Operator(eq, subs=v.grid.spacing_map)

# Initial conditions
v.data[:] = 0
for i in range(shape[0]):
    v.data[0, i] = max((smin + ds0 * i) - K, 0)
```
The next cell fot the run
```
startDevito = timer.time()
    
# Apply operator
op.apply(dt=dt0)

endDevito = timer.time()
```

Reply:
Added, thanks for the clarification